.github/workflows/generate.yml

```diff
@@ -49,10 +46,82 @@ jobs:
             else
               echo "No change in auto generated go snappi"
           fi
-          echo "::set-output name=sha::$(git rev-parse HEAD)"
-      - name: Run tests
+          echo "::set-output name=sha::$(git rev-parse HEAD)"      
+      - name: Archive generated artifacts
+        uses: actions/upload-artifact@v2
+        with:
+          name: generated-artifacts
+          path: |
+            dist
+            snappi
+            gosnppi
+
+  python_tests:
+    needs: build
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 1
+      matrix:
+        python-version: [2.7, 3.6]
```

Comment:
let's use all py versions since 3.6 to 3.10 (put 3.10 in quotes)

Reply:
add 
```
python-version: [2.7, 3.6, 3.7, 3.8, 3.9, "3.10"]
```